clients/fides-js/src/lib/i18n/locales/en/messages.json

```diff
@@ -0,0 +1,3 @@
+{
+  "experience.title": "Title in English"
```

Comment:
TODO: update with real message catalog.

Note that I won't actually have `experience.*` strings here, that prefix will be used for the dynamic strings.